tensorflow/core/kernels/mkl_fused_batch_norm_op.cc

```diff
@@ -843,13 +873,17 @@ class MklFusedBatchNormGradOp : public OpKernel {
       const size_t kScaleIndex = 2;     // index of scale tensor
       const size_t kMeanIndex = 3;      // index of saved_mean tensor
       const size_t kVarianceIndex = 4;  // index of saved_variance tensor
+      const size_t kReSpaceIndex = 5;   // index of reserved space 3 tensor
```

Comment:
Please type the name in full for readability.
```suggestion
      const size_t kReservedSpaceIndex = 5;   // index of reserved space 3 tensor
```